app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -352,7 +357,17 @@ class SearchFragment : Fragment(), UserInteractionHandler {
         }
     }
 
+    private fun animateBackButtonAway() {
+        val xTranslation =
+            requireView().back_button.width.toFloat() * BACK_BUTTON_ANIMATION_WIDTH_MULTIPLIER
```

Comment:
It looks like this constant is to make up for padding + margin?  That works, but it's a little bit messy, opens us up to weird bugs if we ever adjust either of those values, and requires a reader to figure out what it's handling.  What about something like this:

```kotlin
val backButton = requireView().back_button
val xTranslation = with(backButton) {
    -(width + marginStart + paddingStart).toFloat()
}
```

Reply:
Seems reasonable to me!